src/addons/Portal/Portal.js

```diff
@@ -28,8 +29,22 @@ class Portal extends Component {
     /** Additional classes. */
     className: PropTypes.string,
 
+    /**
+     * Controls whether or not the portal should close on a click on the portal background.
+     * NOTE: This differs from closeOnDocumentClick:
+     * - DocumentClick - any click not within the portal
+     * - BackgroundClick - a click not within the portal but within the portal's wrapper
+     */
+    closeOnBackgroundClick: customPropTypes.every([
```

Comment:
If I'm reading corretly, the "Background" is the Dimmer when using a Modal with a Dimmer.  Though, this prop name wouldn't apply to say, a Popup, or some other component.  If so, what about using something like `closeOnRootNodeClick` instead?